packages/playground/wordpress/src/index.ts

```diff
@@ -160,19 +160,9 @@ export async function setupPlatformLevelMuPlugins(php: UniversalPHP) {
 			 * Reload page to ensure the user is logged in correctly.
 			 * WordPress uses cookies to determine if the user is logged in,
 			 * so we need to reload the page to ensure the cookies are set.
-			 *
-			 * Both WordPress home url and REQUEST_URI may include the site scope
-			 * subdirectory.
-			 * To prevent the redirect from including two scopes, we need to
-			 * remove the scope from the REQUEST_URI if it's present.
 			 */
-			$redirect_url = $_SERVER['REQUEST_URI'];
-			if (strpos($redirect_url, '/scope:') === 0) {
-				$parts = explode('/', $redirect_url);
-				$redirect_url = '/' . implode('/', array_slice($parts, 2));
-			}
```

Comment:
We don't need the workaround for login anymore

Reply:
Do we have any E2E tests to confirm this still works?

Reply:
Yes, we do. Specifically[ /wp-admin/ redirect tests](https://github.com/WordPress/wordpress-playground/blob/40a45aad96b46edd8a5d566730b8a6255c3bb492/packages/playground/website/playwright/e2e/query-api.spec.ts#L95-L105) would fail in Safari if the fix didn't work.